app/split_integer.py

```diff
@@ -1,7 +1,11 @@
 def split_integer(value: int, number_of_parts: int) -> list:
     parts = []
+    number = value
     for parts_left in range(number_of_parts, 0, -1):
-        next_number = value // parts_left
-        parts.append(value // parts_left)
-        value -= next_number
+        next_number = number // parts_left
```

Comment:
Using integer division (`//`) here may not distribute the value evenly across all parts, especially if `value` is not perfectly divisible by `number_of_parts`. Consider adjusting the logic to handle the remainder.